endersgame/attackers/__init__.py

```diff
@@ -0,0 +1,3 @@
+UP = 1.
+DOWN = -1.
+NONE = 0.
```

Comment:
todo: do not use floats

Reply:
Actually we should encourage floats. See https://github.com/microprediction/endersgame/blob/main/endersgame/attackers/calibratedattacker.py